pkg/controller/admissionchecks/provisioning/constants.go

```diff
@@ -17,10 +17,12 @@ limitations under the License.
 package provisioning
 
 const (
-	ConfigKind             = "ProvisioningRequestConfig"
-	ControllerName         = "kueue.x-k8s.io/provisioning-request"
-	ConsumesAnnotationKey  = "cluster-autoscaler.kubernetes.io/consume-provisioning-request"
-	ClassNameAnnotationKey = "cluster-autoscaler.kubernetes.io/provisioning-class-name"
+	ConfigKind                       = "ProvisioningRequestConfig"
+	ControllerName                   = "kueue.x-k8s.io/provisioning-request"
+	DeprecatedConsumesAnnotationKey  = "cluster-autoscaler.kubernetes.io/consume-provisioning-request"
+	DeprecatedClassNameAnnotationKey = "cluster-autoscaler.kubernetes.io/provisioning-class-name"
```

Comment:
Is there any timeline for disabling these deprecated annotations in the CA side?
cc: @yaroslava-serdiuk

Reply:
I think we can remove deprecated annotation in next release, which will happen in [the end of August](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md#schedule).
cc: @mwielgus

Reply:
@yaroslava-serdiuk Thank you for letting me know!
@PBundyra, Could you open a dedicated issue on the Kueue side so that we can discuss which ClusterAutoscaler versions should be supported in the Kueue and when we should stop using the deprecated annotations on the Kueue side?

Reply:
From k8s deprecation policy: `Deprecated behaviors must function for no less than 1 year after their announced deprecation.`

Reply:
That makes sense. Let us try to keep this deprecation in https://github.com/kubernetes-sigs/kueue/issues/2747.